install/kubernetes/experimental-install.yaml

```diff
@@ -172,77 +170,14 @@ data:
   config.yaml: |
     peer-service: unix:///var/run/cilium/hubble.sock
     listen-address: :4245
+    dial-timeout: 
+    retry-timeout: 
+    sort-buffer-len-max: 
+    sort-buffer-drain-timeout: 
     disable-client-tls: true
     disable-server-tls: true
 ---
-# Source: cilium/charts/hubble-ui/templates/configmap.yaml
```

Comment:
It looks like the hubble-ui is being disabled in experimental-install, but the whole idea of the experimental-install is to enable all of the fancy features so that users can deploy one YAML and see all the goodness.

Was this intentional? I think we should re-add hubble UI to this chart (presumably by passing the relevant arguments on the CLI in the `install/kubernetes/Makefile`).